.github/workflows/tests-yaml-lint.yml

```diff
@@ -0,0 +1,11 @@
+name: Yaml Lint
```

Comment:
I think it's preferred to stylize YAML in upper-case rather than Yaml.

Reply:
Changed.